include/nlohmann/thirdparty/hedley/hedley.hpp

```diff
@@ -1,15 +1,16 @@
 #pragma once
 
+//     __ _____ _____ _____
+//  __|  |   __|     |   | |  JSON for Modern C++
+// |  |  |__   |  |  | | | |  version 3.10.5
+// |_____|_____|_____|_|___|  https://github.com/nlohmann/json
+//
+// SPDX-FileCopyrightText: 2013-2022 Niels Lohmann <https://nlohmann.me>
+// SPDX-FileCopyrightText: 2016-2021 Evan Nemerson <evan@nemerson.com>
+// SPDX-License-Identifier: MIT
+
```

Comment:
This file seems to be under CC0-1.0 and (C) Evan Nemerson. The only change is the rename from `HEDLEY_*` to `JSON_HEDLEY_*`?

Reply:
Yes, the only change is the renaming. The result is licensed as MIT to avoid confusion.

Reply:
IANAL, but …

The re-licensing seems fine. Are you sure the copyright assignments are proper, though?

Evan has disclaimed his copyright, which in the US, I believe, he can do. Other jurisdictions (i.e., Germany) do not recognize this. So, in this case, CC0 applies and Evan is the copyright holder under german law.

Under US copyright law, adding a prefix to some names isn't considered transformative, i.e., doesn't "add something new for a different purpose or of a different character". I imagine german copyright law equally requires something more for a work to be copyrightable.

Copyright in an open-source project seems like such a nightmare, but for this file at least, it seems pretty clear.

I'll stop now. :-)